src/client/components/lazy-chat/lazy-chat.vue

```diff
@@ -74,4 +74,62 @@ export default {
 .lazy-chat__prompt {
   box-shadow: var(--shadow-wide-grey);
 }
+
+.lazy-chat__open-button {
+  width: 54px;
+  height: 54px;
+  border-radius: 50%;
+  padding: 0;
+  margin-right: -1px;
+  margin-bottom: -1px;
+  position: relative;
+}
+
+.lazy-chat__open-button:before {
+  content: '';
+  display: inline-block;
+  margin-left: 0;
+  margin-bottom: -6px;
+  width: 32px;
+  height: 26px;
+  background-image: url("/images/chat-bubble.svg");
+  background-position: center !important;
+  background-size: contain !important;
+  background-repeat: no-repeat !important;
+}
+
+.loading:before {
+  animation: button__spinner 2s linear infinite;
+  background-image: url('/images/spinner.png');
+  margin-bottom: 0 !important;
+}
+
+@keyframes button__spinner {
+  0% { transform: rotate(0deg) }
+  100% { transform: rotate(360deg) }
+}
+
+#crisp-chatbox[data-last-operator-face="false"] span[data-is-ongoing="false"] {
```

Comment:
I suppose the `!important` here and for the other `crisp-chatbox` are needed since we don't control the crisp-chatbox. I think you should document that and maybe move this styling into a separate css-only component, or some other 'vendor-overrides' file, because they tend to become confusing and unmaintained when they are part of one of our own components